client.go

```diff
@@ -82,22 +83,34 @@ func NewDgraphClient(clients ...api.DgraphClient) *Dgraph {
 //		defer conn.Close()
 //		dgraphClient := dgo.NewDgraphClient(api.NewDgraphClient(conn))
 func DialCloud(endpoint, key string) (*grpc.ClientConn, error) {
-	u, err := url.Parse(endpoint)
-	if err != nil {
-		return nil, err
+	var grpcHost string
+	switch {
+	case strings.Contains(endpoint, ".grpc.") && strings.Contains(endpoint, ":"+cloudPort):
+		grpcHost = endpoint
+	case strings.Contains(endpoint, ".grpc.") && !strings.Contains(endpoint, ":"+cloudPort):
```

Comment:
Will cloud port always be 443? It can also be 21 for http right?

Reply:
you probably mean 80 and no, it will always be 443.